docs: add docs about API, create new pages (#56)
microsoft/griffel: apps/website/docs/react/api/make-static-styles.md

---
sidebar_position: 3
---

import OutputTitle from '@site/src/components/OutputTitle'

# makeStaticStyles

> Creates styles attached to a global selector. This is especially useful for CSS resets, for example [normalize.css](https://github.com/necolas/normalize.css/).

Reviewer: ```suggestion
Creates styles with a global selector. This is especially useful for CSS resets, for example [normalize.css](https://github.com/necolas/normalize.css/).
```
